scvi/module/base/_base_module.py

```diff
@@ -348,6 +348,8 @@ class BaseLatentModeModuleClass(BaseModuleClass):
     @property
     def latent_data_type(self) -> Optional[LatentDataType]:
         """The latent data type associated with this module."""
+        if not hasattr(self, "_latent_data_type"):
```

Comment:
Is it common practice to set a property in a getter method? How about just returning `None` if `not hasattr(self, "_latent_data_type")`?

Reply:
Done.